CHANGELOG.md

```diff
@@ -2,7 +2,42 @@
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
-## [Unreleased]
+## [5.1.0]
+
+### Added
+- Support for reading multiple sheets from an Excel file simultaneously
+- Enable exporting of multiple feature types to Excel
+- Allow loading Excel with blank rows and sheets for schema and source data
+- Add predefined and custom formatters for Date cells in Excel import
+- Enable exporting feature types with no data in Excel
+- Align Excel with CSV export wizard
+- Prefer properties with short paths as GeoJSON geometry property
+- Add wizard page to select the read mode for GeoJSON/JSON
+- Extend JSON reader functionality
```

Comment:
There was no Json reader before, so that should be mentioned as new feature.
Small pieces of the feature or fixes that were done to it before release should not be mentioned.

Reply:
Could be enough to keep this 2:
- Implement JSON reader functionality
- Support loading schema from JSON data